### ISSUE TITLE
Pose estimation tutorial has a wrong function

### ISSUE THREAD

**Issue by albertoZurini** [opencv/opencv#26692]
### Describe the doc issue

Hello! This piece of code: [https://github.com/opencv/opencv/blob/4d26e16af8f45dc8d356770749ae273c8992aa4e/doc/py_tutorials/py_calib3d/py_pose/py_pose.markdown?plain=1#L38-L43](https://github.com/opencv/opencv/blob/4d26e16af8f45dc8d356770749ae273c8992aa4e/doc/py_tutorials/py_calib3d/py_pose/py_pose.markdown?plain=1#L38-L43)

### Fix suggestion

Should be changed as follows:
```python
def draw(img, corners, imgpts):
    corner = tuple(corners[0].ravel().astype("int32"))
    imgpts = imgpts.astype("int32")
    img = cv.line(img, corner, tuple(imgpts[0].ravel()), (255,0,0), 5)
    img = cv.line(img, corner, tuple(imgpts[1].ravel()), (0,255,0), 5)
    img = cv.line(img, corner, tuple(imgpts[2].ravel()), (0,0,255), 5)
    return img
```

This is because `cv.lline` expects the points to be integers and not floats.

Can I submit a PR?

PoC: [https://github.com/albertoZurini/UNITS-ComputerVision/blob/master/1_Photos/3_pose_estimation.ipynb](https://github.com/albertoZurini/UNITS-ComputerVision/blob/master/1_Photos/3_pose_estimation.ipynb)

**Comment by s-trinh** [comment]
> Can I submit a PR?

Yes you can, see https://github.com/opencv/opencv/wiki/How_to_contribute for more information

Basically:
- fork the repo
- add a new remote, something like this: `git remote add upstream https://github.com/opencv/opencv.git` for sync
- create a new branch from `4.x`
- check your name/email are correctly set: `git config user.email` / `git config user.name`
- add your changes, commit and push to your `origin`
- and open your merge request
